### PR TITLE
fix(error): invalid return on `IsOtpError`

### PR DESCRIPTION
**Pull request by ipfaze**
Fix https://github.com/Scalingo/cli/issues/1029

- [x] Add a [changelog entry](https://changelog.scalingo.com/)

### CODE REVIEW

File: http/errors.go [hunk 1]
```diff
@@ -9,6 +9,7 @@ import (
 	"gopkg.in/errgo.v1"
 
 	"github.com/Scalingo/go-scalingo/v6/debug"
+	scErrors "github.com/Scalingo/go-utils/errors/v2"
```
Comment (EtienneM) [inline http/errors.go:12]: issue: in order to match Scalingo convention

```suggestion
	errorspkg "github.com/Scalingo/go-utils/errors/v2"
```

File: http/errors_test.go [hunk 2]
```diff
@@ -0,0 +1,55 @@
+package http
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_IsOTPRequired(t *testing.T) {
```
Comment (EtienneM) [inline http/errors_test.go:10]: suggestion(non blocking): it really looks like this test would be better with table tests

**Review by EtienneM — COMMENTED** [review]
praise: thanks for taking over this issue :pray: 

question: did you tested it in the CLI to ensure it fixes the issue?

File: http/errors_test.go (hunk 2 above)
Comment (curzolapierre) [reply http/errors_test.go:10]: Yep why not

**Comment by curzolapierre** [comment]
@EtienneM Yes I've tested with the CLI, it solves the related issue

**Review by EtienneM — APPROVED** [review]
:tada:

**Pull request merged**